src/web.rs

```diff
@@ -46,6 +47,18 @@ pub struct WebImpl {
     width: u32,
 }
 
+pub enum Canvas {
+    Canvas {
+        canvas: HtmlCanvasElement,
+        ctx: CanvasRenderingContext2d,
+    },
+    #[cfg(web_sys_unstable_apis)]
```

Comment:
Not exactly ideal, but it works and doesn't really affect the public API if a user doesn't use `cfg(web_sys_unstable_apis)`.
As ugly as this is, it's how `wasm-bindgen` works.

We can remove this in the next `web-sys` release (https://github.com/rustwasm/wasm-bindgen/pull/3416) or wait another turn for it, WDYT @notgull?

Reply:
I'd say to wait for it, since the next release shouldn't be that far away.

Reply:
Alright then.